doc/source/user-storage.md

```diff
@@ -220,3 +220,25 @@ kubectl --namespace=<your-namespace> delete pvc <pvc-name>
 
 Remember that deleting someone's `PVC`s will delete all their data, so do so
 with caution!
+
+## Additional storage volumes
+
+If you already have a `PersistentVolume` and `PersistentVolumeClaim` created
+outside of JupyterHub you can mount them inside the user pods.
+For example, if you have a shared `PersistentVolumeClaim` called
+`jupyterhub-shared-volume` you could mount it in all user pods:
+
+```yaml
+singleuser:
+  storage:
+    extraVolumes:
+      - name: jupyterhub-shared
+        persistentVolumeClaim:
+          claimName: jupyterhub-shared-volume
+    extraVolumeMounts:
+      - name: jupyterhub-shared
+        mountPath: /home/jovyan/shared
```

Comment:
@manics :heart: a documentation PR! I'm growing more and more fond of documentation as I have consumed so much lately.

---

I think this mount may fail in some way, as the `mountPath` is within another mount's path. If I recall correctly what I've read but not tested myself. Can one nest volumeMounts?

Perhaps we could also reference [k8s docs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#volumemount-v1-core) for volumeMounts and/or provide a note about `subPath` for entries in `extraVolumeMounts` while we are at it.

Reply:
If it is possible to nest volumes like this, it LGTM!

Reply:
Good point, on my system there aren't any per-user volumes. How does `/home/shared` sound?

Reply:
Sounds great @manics !

Reply:
Updated!